sdk/burdock/query/sql/metadata/config_file.py

```diff
@@ -1,23 +1,29 @@
 import yaml
 from .metadata import *
-
+from io import StringIO
```

Comment:
Let's not change this file.  The way to create in-memory metadata is with object literals.

Reply:
FYI, here is a code sample for making metadata with object literals:

from burdock.query.sql.metadata.metadata import *
import copy

table1 = Table("dbo", "d1", 5000, \
    [\
        String("DeviceID", 0, True),\
        Boolean("Refurbished"), \
        Float("Temperature", 20.0, 70.0)
    ])

table2 = copy.copy(table1)
table2.name = "d2"

x = Database([table1],"csv")
y = Database([table2],"csv")

print(x)
print(y)

Reply:
(With formatting)

`from burdock.query.sql.metadata.metadata import *
import copy

table1 = Table("dbo", "d1", 5000, \
    [\
        String("DeviceID", 0, True),\
        Boolean("Refurbished"), \
        Float("Temperature", 20.0, 70.0)
    ])

table2 = copy.copy(table1)
table2.name = "d2"

x = Database([table1],"csv")
y = Database([table2],"csv")

print(x)
print(y)`

Reply:
`from burdock.query.sql.metadata.metadata import *
import copy

table1 = Table("dbo", "d1", 5000, \
    [\
        String("DeviceID", 0, True),\
        Boolean("Refurbished"), \
        Float("Temperature", 20.0, 70.0)
    ])

table2 = copy.copy(table1)
table2.name = "d2"

x = Database([table1],"csv")
y = Database([table2],"csv")

print(x)
print(y)`

Reply:
Sure. Removed the changes made.